src/vaillant_netatmo_api/thermostat.py

```diff
@@ -385,8 +386,10 @@ def __init__(
         self.battery_percent = battery_percent
         self.setpoint_away = Setpoint(**setpoint_away)
         self.setpoint_manual = Setpoint(**setpoint_manual)
-        self.therm_program_list = [Program(**program) for program in therm_program_list]
+        self.therm_program_list = [
+            Program(**program) for program in therm_program_list]
         self.measured = Measured(**measured)
+        self.energy_usage = EnergyUsage(**energy_usage)
```

Comment:
I suggest `energy_usage` is removed from `Thermostat` class completely. This class is a result of a `getthermostatsdata` call and that call doesn't have this field.

Reply:
@MislavMandaric I removed EnergyUsage attribute as you requested